apm-agent-plugins/apm-spring-webflux/apm-spring-webclient-plugin/src/main/java/co/elastic/apm/agent/springwebclient/WebClientSubscriber.java

```diff
@@ -192,5 +193,19 @@ private void cancelSpan() {
             debugTrace(false, "cancelSpan", span);
         }
     }
+    @SuppressWarnings("unchecked")
```

Comment:
[For Reviewer] This is suddenly required because onNext doesn't declare checked exceptions anymore. For backwards compatibility reasons, I figured it would be best to still continue throwing unwrapped exceptions.

Reply:
May be worth adding a comment to make that a bit more explicit and document which version introduced it.